apis/storage/v1beta1/bucket_types.go

```diff
@@ -55,41 +46,38 @@ type S3BucketParameters struct {
 	// provisioning.
 	// +kubebuilder:validation:Enum=Read;Write;ReadWrite
 	LocalPermission *storagev1alpha1.LocalPermissionType `json:"localPermission"`
```

Comment:
`LocalPermission` doesn't seem to be included in any call. I'd consider removing it for now until we get to implement those other subresources.